sensor.go

```diff
@@ -82,7 +83,7 @@ func InitSensor(options *Options) {
 	sensor = newSensor(options)
 
 	// enable auto-profiling
-	if options.EnableAutoProfile {
+	if options.EnableAutoProfile || strings.ToLower(os.Getenv("INSTANA_AUTO_PROFILE")) == "true" {
```

Comment:
I think we could simplify configuration and simply check whether `INSTANA_AUTO_PROFILE` is set, just like we do with `INSTANA_DEBUG`.